corehq/apps/accounting/tasks.py

```diff
@@ -844,7 +845,7 @@ def _apply_downgrade_process(oldest_unpaid_invoice, total, today, subscription=N
         })
 
     days_ago = (today - oldest_unpaid_invoice.date_due).days
-    if days_ago == 61:
+    if days_ago >= 61:
```

Comment:
Is this likely to cause many domains to be suddenly downgraded? Is the sales team ready for the storm?

Reply:
many domains?...likely not. according to erin, she has been the one in charge of adding / removing `skip_autodowngrade` on projects and is already in tight communication with those projects. There's less than a handful who will likely be downgraded immediately.